File: Easy/22.MeanMode/solve.rb
require "set"

def getMode(arr)
	sortedSet = SortedSet.new(arr)
	mode = max_occurences = 0
	sortedSet.each do |n|
		if arr.count(n) > max_occurences
			max_occurences = arr.count(n)
			mode = n
		end
	end
	mode
end

def getMean(arr)
	arr.reduce(:+) / arr.length
end

def MeanMode(arr)
	  getMode(arr) == getMean(arr) ? 1:0 
end
         
# keep this function call here    
puts MeanMode(STDIN.gets)

File: Medium/32.StockPicker/solve.py
def StockPicker(arr):
	best = 0
	for i in range(0, len(arr)-1):
		for j in range(i+1, len(arr)):
			if arr[j] - arr[i] > best:
				best = arr[j] - arr[i]
	if best > 0:
		return best
	return - 1
    
# keep this function call here  
print StockPicker(raw_input())